lte/gateway/c/core/oai/test/mme_app_task/test_mme_procedures.cpp

```diff
@@ -1828,7 +1828,7 @@ TEST_F(
   EXPECT_EQ(mme_state_p->nb_default_eps_bearers, 0);
   EXPECT_EQ(mme_state_p->nb_ue_idle, 0);
 }
-
+#if 0
```

Comment:
why is this test excluded?

Reply:
I this this was on an intermediate PR. I don't see it in the file changes